File: 20180709/ArrayPartitionI.java
class Solution {
    public int arrayPairSum(int[] nums) {
        int n = nums.length;
        Arrays.sort(nums);
        int res = 0;
        for(int i = 0; i < n; i ++){
            if(i % 2 == 0){
                res += nums[i];
            }
        }
        return res;
    }

}

File: 20180709/ArrayPartitionI.py
class Solution(object):
    def arrayPairSum(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        n = len(nums)
        nums.sort()
        res = [nums[i] for i in range(n) if i%2 == 0]
        return sum(res)

            